editoast/src/views/train_schedule/mod.rs

```diff
@@ -66,17 +64,13 @@ pub enum TrainScheduleError {
 }
 
 pub fn routes() -> impl HttpServiceFactory {
-    web::scope("/train_schedule/{id}").service((
-        get,
-        patch,
-        standalone_simulation,
-        get_result,
-        get_results,
-    ))
+    web::scope("/train_schedule")
+        .service(get_results)
+        .service(scope("/{id}").service((get, patch, standalone_simulation, get_result)))
 }
 
 /// Return a specific timetable with its associated schedules
-#[get("")]
+#[get("/{id}")]
```

Comment:
I think it's not necessary to add the `{id}` here since it's induced by the `scope()` above.